daemons/dss-index/app.py

```diff
@@ -1,4 +1,6 @@
-import os, sys, json, boto3, domovoi
+import os, sys, json, boto3, logging
```

Comment:
Normally I like to group imports into three groups (stdlib, third-party, local/relative) but within those groups, I have multiple imports per line for brevity. Here I accidentally put a third-party import (`domovoi`) among the stdlib imports so I'm breaking it out.